### ISSUE TITLE
Suggestion: Add "Install Docker" to the "Install Minikube" page (macOS Catalina)

### ISSUE THREAD

**Issue by pedz** [kubernetes/minikube#6313]
TL; DR - I needed to install Docker and start it up but the [install page](https://kubernetes.io/docs/tasks/tools/install-minikube/) doesn't explain that to me.

I'm not sure if this repo is the right place for this issue.  I don't know if Minikube depends upon Docker or if kubectl depends upon Docker or if I'm misinterpreting my experience and there is / was something else wrong with my environment.  I don't see Docker listed as a requirement in the places I've looked.  It may be drop dead obvious but I'm an old dog trying to learn new tricks.

**The exact command to reproduce the issue**:

    brew install minikube
    minikube start --vm-driver=hyperkit

**The full output of the command that failed**:<details>

😄  minikube v1.6.2 on Darwin 10.15.2
✨  Selecting 'hyperkit' driver from user configuration (alternates: [virtualbox vmwarefusion])
🔥  Creating hyperkit VM (CPUs=2, Memory=2000MB, Disk=20000MB) ...

💣  Unable to start VM. Please investigate and run 'minikube delete' if possible
❌  Error: [HYPERKIT_CRASHED] create: Error creating machine: Error in driver during machine creation: hyperkit crashed! command line:
  hyperkit loglevel=3 console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 systemd.legacy_systemd_cgroup_controller=yes random.trust_cpu=on hw_rng_model=virtio base host=minikube
💡  Suggestion: Hyperkit is broken. Upgrade to the latest hyperkit version and/or Docker for Desktop. Alternatively, you may choose an alternate --vm-driver
⁉️   Related issues:
    ▪ https://github.com/kubernetes/minikube/issues/6079
    ▪ https://github.com/kubernetes/minikube/issues/5780

😿  If the above advice does not help, please let us know: 
👉  https://github.com/kubernetes/minikube/issues/new/choose

</details>

**The output of the `minikube logs` command**:<details>

💣  command runner
❌  Error: [SSH_AUTH_FAILURE] getting ssh client for bootstrapper: Error dialing tcp via ssh client: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain
💡  Suggestion: Your host is failing to route packets to the minikube VM. If you have VPN software, try turning it off or configuring it so that it does not re-route traffic to the VM IP. If not, check your VM environment routing options.
📘  Documentation: https://minikube.sigs.k8s.io/docs/reference/networking/vpn/
⁉️   Related issues:
    ▪ https://github.com/kubernetes/minikube/issues/3930

😿  If the above advice does not help, please let us know: 
👉  https://github.com/kubernetes/minikube/issues/new/choose

</details>

**The operating system version**:

macOS Catalina 10.15.2

**Comment by afbjorklund** [comment]
Minikube does not technically depend on Docker, but it is an easy workaround to fix HyperKit installations (since HyperKit is a Docker product) and the `docker` client might be useful for building...

Preferrably the macOS hypervisor that minikube uses would work out-of-the-box (whether `hyperkit` or something else based on Hypervisor.framework) and that building images would work without it.

**Comment by afbjorklund** [comment]
I think this would fix the _client_:

```
brew install docker
eval $(minikube docker-env)
```

It would be nice to fix the driver...

**Comment by priyawadhwa** [comment]
Hey @pedz, I'm going to close this issue as a dupe of #6402. We'll be looking into improving documentation in the coming weeks, so you can check for updates on this there. If you'd be interested in contributing this documentation, please let me know! Thank you.
